integration_framework/views.py

```diff
@@ -1425,7 +1425,8 @@ def get_json_protocol_data(pk):
                         except Exception as e:
                             pass
                     count += 1
-
+        if val.strip() == '-':
```

Comment:
Давай ещё пробелы пофиксим. Я подумал надо защититься от потенциальных косяков.
То есть вот так:
```suggestion
        if val.strip() in ('-', ''):
```

Reply:
Это на случай, если значение равно пробелу